### DISCUSSION TITLE
Why the same search  time cost has a big gap ?

### DISCUSSION THREAD

**Discussion by TouSS** [JanusGraph/janusgraph#3014] [Q&A] [answered]
HI，

I have two search like this:
g.V(id1).out().count().next()
g.V(id2).out().count().next()

I got a big gap time cost for the two search， Why?

Anyone can help me?

**Comment by FlorianHockmann** [comment]
How did you execute these traversals? Via Gremlin Console? And what are the results of the two traversals?

**Comment by TouSS** [comment, marked as answer]
vertex，id2 which has 10w edges, id1 only 10 edges,  so has a big gap time cost.

**Reply by li-boxuan** [reply]
This makes sense. `g.V(id2).out()` visits edges 10 thousand times more than `g.V(id1).out()` does, so it takes much more time.
